src/argilla/training/spacy.py

```diff
@@ -30,9 +30,6 @@ class ArgillaSpaCyTrainer:
     _logger = logging.getLogger("ArgillaSpaCyTrainer")
     _logger.setLevel(logging.INFO)
 
-    require_version("torch")
```

Comment:
I believe It is also possible to use CPU to train with spacy-transformers and therefore torch.

Reply:
Hmm so it's weird because the train configuration when selecting the GPU is generated just for `spacy-transformers`, but I've tested it and it also works fine with CPU. Same thing for training `spaCy` models, so either we temporarily drop support for `spacy-transformers` models or just figure out the best way to combine both.